lbz/configuration.py

```diff
@@ -0,0 +1,45 @@
+from abc import abstractmethod
+from os import getenv
+from typing import Any, Callable, Generic, Optional, TypeVar
+
+from lbz.aws_boto3 import SSM
+
+T = TypeVar("T")
+
+
+class BaseValue(Generic[T]):
+    def __init__(
+        self,
+        key: str,
+        parser: Optional[Callable[..., T]] = None,
```

Comment:
I think every parser should be exactly the same, accepting string as a single positional argument 🤔

Reply:
That's more tricky actually...

Reply:
As it turned out, it wasn't that tricky but in the newest version, it should accept `Any` 🙂